src/main/java/training/day2/Main.java

```diff
@@ -104,8 +154,40 @@ public static void main(String[] args) {
         for (String day1 : days) {
             System.out.println(day1);
         }
+*/
+        //Homework
+        //Zadacha 1
+        String[] words = {"Алфа", "аватар", "беда", "бреза", "салон"};
+        for (String word1 : words){
+            if (word1.charAt(0) == 'a' || word1.charAt(0) == 'A'){
```

Comment:
Since the words are on cyrillic, the comparison between charAt() and a value must be a cyrillic as well.